### DISCUSSION TITLE
Default "program file" when Play is pressed

### DISCUSSION THREAD

**Discussion by davidyang** [pybricks/support#1901] [Ideas]
My team is putting one file per mission model and one issue is that they often hit play from their mission model file instead of the main file that loads all of them.  Any possibility of adding a "default" Play program file?

**Comment by laurensvalk** [comment]
Thank you. This could indeed be a nice feature.

Depending on what you'd like to do, you could also ensure that each mission is both independently testable and usable within the main program. This would allow the kids to test each mission quickly and combine them later.

I've turned this into an actionable issue here https://github.com/pybricks/support/issues/1903. Feel free to discuss further there or add other ideas. Thanks!
